src/routes/articles.js

```diff
@@ -42,7 +49,32 @@ const createArticle = async (req, res, next) => {
 		next(err)
 	}
 }
+// CREATE A POST COMMENT
 
+const createComments = async (req, res, next) => {
```

Comment:
another case of using plural of 1createComments` here because the `createComment`  name is taken @beblicarl. Kindly update this name to reflect what the function does. 

If you don't want to worry about finding another name for it, you can go with the option of using the service directly without destructuring

```js
const postService = require('../services/posts');

// and use it like this 
const {post, insertedComment} = await postService.createComment({
			userId, 
			id, 
			comment
})
}
```

Reply:
noted. I have made the changes now @jocrah